### PR TITLE
`impl fmt::Display`

### PR DESCRIPTION
**Pull request by Dillon-Roller**

### CODE REVIEW

**Comment by Dillon-Roller** [comment]
this gives us `to_string()` for free on `Player` and `Move`

File: src/moves.rs [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+
+impl fmt::Display for Move {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let files = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'];
+        let ranks = ['1', '2', '3', '4', '5', '6', '7', '8'];
+
+        let from_file = files[self.from.x];
+        let to_file = files[self.to.x];
+
+        let from_rank = ranks[8 - self.from.y - 1];
```
Comment (DanielAlbl) [inline src/moves.rs:18]: I don't think y needs to be reversed.

File: src/moves.rs [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+        let to_rank = ranks[8 - self.to.y - 1];
+
+        write!(f, "{}{}->{}{}", from_file, from_rank, to_file, to_rank)
```
Comment (DanielAlbl) [inline src/moves.rs:21]: I wonder if it could be possible to print chess emojis lol https://www.i2symbol.com/symbols/chess.

**Review by DanielAlbl — COMMENTED** [review]
Hell yeah

File: src/board.rs [hunk 3]
```diff
@@ -122,6 +122,8 @@ impl Board {
             piece = Piece::Queen(self.player)
         }
         self.squares[mv.to.y][mv.to.x] = Some(piece);
+        println!("{} : {}", piece, mv);
```
Comment (DanielAlbl) [inline src/board.rs:125]: Do you think this should be in game `move_piece` or baord `move_piece`?

**Review by DanielAlbl — APPROVED** [review]

**Comment by Dillon-Roller** [comment]
This errors when you move a piece

**Pull request merged**